dangerfile.ts

```diff
@@ -149,10 +149,11 @@ const reactRecogniseProp = ["React does not recognize"]
 const invalidTags = ["The tag <rect>", "The tag <g>", "The tag <text>"]
 const passingTests = ["✓"]
 const unknownEventHandler = ["Unknown event handler"]
-const propType = ["Failed prop type"]
+const propType = ["Failed prop type", "Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.", 'A props object containing a "key" prop is being spread into JSX:']
 const componentWillReceiveProps = ["componentWillReceiveProps"]
 const invalidCSSProperties = ["is an invalid value for the .* css style property."]
 const invalidProps = ["for a non-boolean attribute", "Invalid ARIA attribute"]
+const actTests = ["inside a test was not wrapped in act(...)"]
```

Comment:
This is a newly introduced warning, tests still pass and there is a few conversations open for this

`https://github.com/reactwg/react-18/discussions/102`

As its not a blocker adding it here now and we should revisit in the future